src/extension.ts

```diff
@@ -51,7 +51,7 @@ export function activate(context: vscode.ExtensionContext) {
               ? folders.length > 1
                 ? `Extension (${folder.name}: ${rel})`
                 : `Extension (${rel})`
-              : 'VS Code Extension',
+              : `VS Code Extension (${folder.name})`,
```

Comment:
Let's only include the name if there's more than one folder in the workspace (like in the above code path)